dev_swift/00_load_data.ipynb

```diff
@@ -60,16 +60,19 @@
    "metadata": {},
```

Comment:
<p>Avoid the use of a `get`-prefix in all names in Swift.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_docs/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Just for our information, why should avoid get in names?</p>

Reply:
See [Swift API Design Guidelines - Naming](https://swift.org/documentation/api-design-guidelines/#naming):

> Name functions and methods according to their side-effects.
> - Those without side-effects should read as noun phrases, e.g. `x.distance(to: y)`, `i.successor()`.
> - Those with side-effects should read as imperative verb phrases, e.g., print(x), x.sort(), x.append(y).
> - Name Mutating/nonmutating method pairs consistently. A mutating method will often have a nonmutating variant with similar semantics, but that returns a new value rather than updating an instance in-place.

`get` is often considered a needless word when those rules apply. Functions that start with `get` or `set` generally do not exist at all in idiomatic Swift libraries. Their smenatics are either generalized by the naming convention or language constructs (e.g. computed property setters).